modules/ROOT/pages/client-id-based-policies.adoc

```diff
@@ -14,48 +14,73 @@ endif::[]
 |500 - Bad response from authorization server, or WSDL SOAP Fault error
 |===
 
-The Client ID Enforcement Policy restricts access to a protected resource, by only allowing HTTP requests if credentials of client applications already xref:api-contracts-landing-page.adoc[registered to (contracted with) the API being targeted] are provided in each request.
+The Client ID Enforcement Policy restricts access to a protected resource, by only allowing HTTP requests if credentials of client applications already xref:api-contracts-landing-page.adoc[registered to (contracted with) the API being targeted] are provided in each request. In Anypoint Platform, the client ID and client secret pair is also known as credentials.
 
```

Comment:
Is there a policy around having the links embedded or in the see more section? Mainly asking about the xref here.

Reply:
Moved it to See Also.